Refine explanation of individual reproduction number (#274)
epiverse-trace/tutorials-middle: episodes/superspreading-estimate.Rmd

Author: to sync with reply above (don't know why GitHub is not capturing the specific inline edits)
```suggestion
- While $R_{0}$ captures the average transmission in the population, the **individual reproduction number** is the random variable representing the expected number of secondary _infections_ caused by a specific _infected_ individual.
```

## The individual reproduction number

> The reproduction number varies at the individual level; that is, the number of secondary infections varies from one infected individual to another. This individual-level count is called the individual reproduction number. The extent to which it differs across individuals is known as the variation in individual reproductive number.

Reviewer: This is a good improvement, but I think that it can be refined further. As defined in Lloyd-Smith et al. (2005): the ‘individual reproductive number’, $\nu$, as a random variable representing the expected number of secondary cases caused by a particular infected individual.

Perhaps it could be stated that the number of secondary cases caused by infectious individuals varies at the individual level. This variation is encapsulated in the individual-level reproduction number.

I think the delineation between 'secondary cases' and 'reproduction number' can be clearer in the tutorials, with the former being related to data and the later being a theoretical/statistical metric that is estimated. For example, I don't think the individual reproduction number should be referred to as a 'count'.

Author: ```suggestion
The reproduction number varies at the individual level; that is, the number of secondary infections varies from one infected individual to another. The statistical metric that captures this individual level variation is called the individual reproduction number. The individual reproduction number is a random variable representing the expected number of secondary infections caused by a particular infected individual.
```

Reviewer: Looks good. Not sure if the wording is a bit similar to be considered plagarism, but I think it's good to be precise so I'd accept this change.

Author: then, I will add citation to paper, as we do it lines above too.

Author: done at https://github.com/epiverse-trace/tutorials-middle/pull/274/commits/7c35f9da07cb14da7e9754f5b320299868d62ad7

> The reproduction number varies at the individual level; that is, the number of secondary infections varies from one infected individual to another. The statistical metric that captures this individual level variation is called the individual reproduction number. The individual reproduction number is a random variable representing the expected number of secondary infections caused by a particular infected individual.

Author: ```suggestion
The reproduction number varies at the individual level; that is, the number of secondary infections varies from one infected individual to another. The statistical metric that captures this individual level variation is called the individual reproduction number. The individual reproduction number is a random variable representing the expected number of secondary infections caused by a particular infected individual ([Lloyd-Smith et al., 2005](https://www.nature.com/articles/nature04153)).
```
